.github/workflows/ci.yml

```diff
@@ -18,32 +18,25 @@ concurrency:
 
 jobs:
   build:
-    name: JDK ${{ matrix.ci_java_version }} / AGP ${{ matrix.ci_agp_version }} / Kotlin ${{ matrix.ci_kotlin_version }}
     runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        ci_java_version: ['20']
```

Comment:
matrix.ci_java_version is still being used in 2 locations. Can you please resolve those references as well?

Reply:
Good catch 07f36eb